package.json

```diff
@@ -6,10 +6,20 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test",
+    "lint": "ng lint",
+    "lf": "ng lint --fix",
+    "prepare": "husky",
+    "docs": "compodoc -p tsconfig.doc.json -s -r 8081",
+    "compodoc:build": "compodoc -p tsconfig.doc.json",
+    "compodoc:build-and-serve": "compodoc -p tsconfig.doc.json -s -r 8081",
+    "compodoc:serve": "compodoc -s"
   },
+  "type": "module",
```

Comment:
As Jordy also noticed, this breaks the `run.js` script. It must be modified to use the `import` syntax, and needs some more changes because it uses `__dirname`.

Reply:
According to the [Angular docs](https://angular.dev/tools/libraries/angular-package-format#esm-declaration), this should have been set already. Strange that `ng new` did not do this.

Reply:
ah sorry, I wasn't aware of the separate run.js file. Should have migrated that, too. Will do.
I think we should do our best to stay away from CommonJs. It's inferior in some technical aspects and codebases typically struggle if the standards are mixed. It was pretty annoying to get some of the eslint stuff working and it was basically only possible via esm.

Reply:
I agree - ES modules are the future and the Angular docs also seem to recommend them.